src/core/dom/index.js

```diff
@@ -566,9 +572,10 @@ export function isServerRendered(element) {
  * @param {HTMLScriptElement} scriptEl
  */
 export function propagateNonce(doc, scriptEl) {
+  /** @type {?HTMLScriptElement} */
   const currentScript = doc.head.querySelector('script[nonce]');
   if (currentScript) {
     const nonce = currentScript.nonce || currentScript.getAttribute('nonce');
-    scriptEl.setAttribute('nonce', nonce);
+    scriptEl.setAttribute('nonce', nonce ?? '');
```

Comment:
This is definitely part of the size difference, especially since `??` gets compiled into something else (which I hate). Passing undefined results in an attribute `nonce="undefined"` (which I also hate). There's a few places like this which are bug-adjacent, albeit probably not causing any user-visible issues. I guess simplest solution is to replace with `||`?